main.yml

```diff
@@ -100,6 +105,7 @@ Resources:
     Type: AWS::CloudFormation::Stack
     Properties:
       Parameters:
+        QueueAddress: !Ref QueueAddress
```

Comment:
I'd alphabetize the list of parameters (putting QueueAdress between KeyPair and Subnet).